src/Cms/ModelWithContent.php

```diff
@@ -735,19 +724,7 @@ public function version(VersionId|string|null $versionId = null): Version
 	 */
 	public function writeContent(array $data, string|null $languageCode = null): bool
 	{
-		$data = $this->contentFileData($data, $languageCode);
-
-		// update the default language, unless a specific language is passed
-		$languageCode ??= 'default';
-
-		try {
-			// we can only update if the version already exists
-			$this->version()->update($data, $languageCode);
-		} catch (NotFoundException) {
-			// otherwise create a new version
-			$this->version()->create($data, $languageCode);
-		}
-
+		$this->version()->save($data, $languageCode ?? 'current', true);
```

Comment:
Isn't the fallback to the current language a breaking change? In such a low level method I'm a bit afraid it will cause regressions TBH.